assets/js/googlesitekit/widgets/register-defaults.js

```diff
@@ -159,4 +168,37 @@ export function registerDefaults( widgetsAPI ) {
 			AREA_DASHBOARD_POPULARITY,
 		],
 	);
+
+	widgetsAPI.registerWidgetArea(
+		AREA_MODULE_SEARCH_CONSOLE_MAIN,
+		{
+			priority: 1,
+			subtitle: '',
```

Comment:
The subtitles can be removed entirely now, since they're optional. I know `''` isn't truthy in JS, but omitting the param seems more explicit than including the empty one.

```suggestion
```